### PR TITLE
Enable handler to handle url in IOS

### PR DESCRIPTION
**Pull request by Ashkan-Oliaie**
-A handler is passed to plugin to enable the url being handled on Flutter side

### CODE REVIEW

File: src/ios/Classes/SwiftSfmcPlugin.swift [hunk 1]
```diff
@@ -5,13 +5,15 @@ import Foundation
 
 public class SwiftSfmcPlugin: NSObject, FlutterPlugin, MarketingCloudSDKURLHandlingDelegate, MarketingCloudSDKEventDelegate {
     
+    static var chanel:FlutterMethodChannel?
```
Comment (sefidgaran) [inline src/ios/Classes/SwiftSfmcPlugin.swift]: channel*

File: src/lib/sfmc_plugin_platform_interface.dart [hunk 2]
```diff
@@ -33,6 +34,9 @@ abstract class SfmcPluginPlatform extends PlatformInterface {
         delayRegistration: delayRegistration);
   }
 
+  setHandler(Future<dynamic> Function(MethodCall call)? handler) =>
```
Comment (sefidgaran) [inline src/lib/sfmc_plugin_platform_interface.dart:37]: please update the example as well. thanks

**Review by sefidgaran — COMMENTED** [review]
Left two comments. thanks

File: src/ios/Classes/SwiftSfmcPlugin.swift (hunk 1 above)
Comment (Ashkan-Oliaie) [reply src/ios/Classes/SwiftSfmcPlugin.swift]: Fixed

File: src/lib/sfmc_plugin_platform_interface.dart (hunk 2 above)
Comment (Ashkan-Oliaie) [reply src/lib/sfmc_plugin_platform_interface.dart:37]: Updated

**Review by sefidgaran — APPROVED** [review]

**Pull request merged**